plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/validators/N4JSFunctionValidator.xtend

```diff
@@ -216,7 +270,7 @@ class N4JSFunctionValidator extends AbstractN4JSDeclarativeValidator {
 		val FunctionFullReport analysis = returnOrThrowAnalysis.exitBehaviourWithFullReport(functionOrFieldAccessor.body?.statements)
 
 		// Dead Code? : Constraints 107
-		holdsNoDeadCode(functionOrFieldAccessor, analysis);
+		// holdsNoDeadCode(functionOrFieldAccessor, analysis);
```

Comment:
why keep this outdated comment? better just remove

Reply:
ok, done